example-app/src/main/java/com/adyen/checkout/example/CheckoutExampleApplication.kt

```diff
@@ -19,7 +19,7 @@ class CheckoutExampleApplication : MultiDexApplication() {
 
     companion object {
         init {
-            Logger.setLogcatLevel(Log.DEBUG)
+            Logger.setLogcatLevel(Log.VERBOSE)
```

Comment:
Is this changed on purpose?

Reply:
good point, see how that SENSITIVE could come in hand if we actually use it 😅

Reply:
hahaha yes 💯